values.yaml

```
@@ -1483,6 +1483,14 @@ connectInject:
       memory: "50Mi"
       cpu: "50m"
 
+  # Sets the failurePolicy for the mutating webhook. By default this will cause pods to fail scheduling while the webhook
```

```suggestion
  # Sets the failurePolicy for the mutating webhook. By default this will cause pods that will be connect-injected to fail scheduling while the webhook
```
Is my suggestion accurate? If so, could be worth specifying.

---

Actually no, it's all pods that are NOT part of the consul install, so agents/servers/gateways/etc are allowed to schedule but nothing else is.

---

@ndhanushkodi - given the above, any hesitations to merge the issue? Thx!

---

can we add something like "By default this will cause pods not part of the consul installation to fail scheduling..."?

But otherwise go ahead and merge!!